tests/test_application_form.py

```diff
@@ -0,0 +1,262 @@
+"""
```

Comment:
This is a massive comment at the beginning of the file - is it necessary?  What's the logic of putting this here over a wiki or something else?

Reply:
Sure I agree - the json schema is tightly coupled with the tests so thought worth having a reference somewhere near the code, in case the schema changes and the tests fail as a result - I can slim down definitely as a first step... But would welcome suggestions on better places for it

Reply:
Schema removed and comment refined